projects/cds/src/cds-models/cds-strategy-request.model.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * A model class that is used when sending a request to the Context-Driven Services Merchandising Strategy service for returning
+ * products for a carousel. There are a number of things that can affect which products are returned, based on the users current
+ * context
+ */
+export interface StrategyRequest {
+  productId?: string;
+  category?: string;
+  facets?: string;
+  date?: string;
+  userId?: string;
```

Comment:
Just an FYI - for security reasons, we should _not_ send the actual user's ID as part of the OCC calls. The back-end supports `current` as the user ID, and it's able to get the actual user ID based on the provided OAuth token.

Reply:
I've removed the user id for now, so I can check what its used for and whether we actually use it. Its worth pointing out though, that the endpoint we are sending this to isn't an OCC endpoint, but a CDS public endpoint that is used by the carousel, so we have no ability to identify the current user

Reply:
Ah, right, I missed that.